Source/WebCore/testing/Internals.cpp

```diff
@@ -2540,7 +2540,7 @@ uint64_t Internals::storageAreaMapCount() const
     if (!page)
         return 0;
 
-    return page->storageNamespaceProvider().localStorageNamespace(page->sessionID()).storageAreaMapCountForTesting();
+    return page->storageNamespaceProvider().localStorageNamespace(page->sessionID(), 5 * 1024 * 1024).storageAreaMapCountForTesting();
```

Comment:
Don't hardcode the value. Use page->settings()->localStorageQuota() instead.

Reply:
Done